tests/functional/tests/bundles/base-yoga.yaml

```diff
@@ -143,15 +143,11 @@ applications:
     charm: ch:nova-compute
     channel: yoga/stable
     num_units: 1
-    storage:
-      ephemeral-device: '40G'
     options:
+      options:
       openstack-origin: *openstack-origin
-      config-flags: auto_assign_floating_ip=False
-      enable-live-migration: false
-      aa-profile-mode: enforce
+      verbose: true
```

Comment:
Did you also want to add `debug: true` here to be consistent with the other bundle? Alternately, did you want to remove the verbose and debug options now that the tests are passing?

Reply:
Yeah I thinki we can remove now